package.json

```diff
@@ -113,4 +111,4 @@
     "react": ">=16.0.0",
     "react-dom": ">=16.0.0"
   }
-}
+}
```

Comment:
`\n` here

Reply:
Weird. I didn't add the the `\n`. Maybe when installed/uninstalled packages it added it.